src/main/java/org/folio/rest/impl/OaiPmhImpl.java

```diff
@@ -53,6 +82,44 @@ public static void init(Handler<AsyncResult<Boolean>> resultHandler) {
     resultHandler.handle(succeededFuture(true));
   }
 
+  @Override
+  public void getOaiVerbs(String verbName, String identifier, String resumptionToken, String from, String until, String set, String metadataPrefix, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
```

Comment:
I think we need to make up a more appropriate name for this method, as it is very confusing.

Reply:
> I think we need to make up a more appropriate name for this method, as it is very confusing.

Okay, what you can suggest for example?